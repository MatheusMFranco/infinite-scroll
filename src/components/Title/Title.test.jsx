import { screen } from '@testing-library/react';
import { renderTheme } from '../../utils/render-theme';
import { Title } from '.';

describe('<Title />', () => {
  it('should render', () => {
    expect.assertions(1);
    renderTheme(<Title>Children</Title>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = renderTheme(<Title>Legacy</Title>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
