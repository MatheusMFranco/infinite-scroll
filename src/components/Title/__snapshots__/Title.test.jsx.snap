// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should match snapshot 1`] = `
.c0 {
  color: #000;
  text-align: center;
  padding-bottom: 1rem;
  font-size: 4.2rem;
}

<h1
  class="c0"
>
  Legacy
</h1>
`;
